mailbox/store/src/test/resources/eml/emailWithNonIndexableAttachment.eml

```diff
@@ -0,0 +1,1655 @@
+Return-Path: <lroyet@linagora.com>
+MIME-Version: 1.0
+Delivered-To: lroyet@linagora.com
+Content-Type: multipart/mixed;
+ boundary="-=Part.0.16b49c355c1c1b54.158b4d6fd1b.bd3741f90275bb0a=-"
+From: Laura ROYET <lroyet@linagora.com>
+Sender: Laura ROYET <lroyet@linagora.com>
+To: Laura ROYET <lroyet@linagora.com>
+Subject: test
+Message-ID: 1480502737913
+Date: Wed, 30 Nov 2016 10:45:38 +0000
+
+---=Part.0.16b49c355c1c1b54.158b4d6fd1b.bd3741f90275bb0a=-
+Content-Type: text/html; name="=?US-ASCII?Q?test.html?="
+Content-Disposition: attachment
+Content-Transfer-Encoding: base64
+
+PCFET0NUWVBFIGh0bWw+CjxodG1sIGNsYXNzPSJtb3ppbGxhIiBsYW5nPSJlbiI+PGhlYWQ+Cjxt
+ZXRhIGh0dHAtZXF1aXY9ImNvbnRlbnQtdHlwZSIgY29udGVudD0idGV4dC9odG1sOyBjaGFyc2V0

```

Comment:
Could we check that we support encoding base64 for indexing ? (that might be the underlying issue, no ?